### ISSUE TITLE
Allow different Personal Identity Number formats on login page

### ISSUE THREAD

**Issue by nikolaykrondev** [ActiveLogin/ActiveLogin.Authentication#56]
Even thought that ActiveLogin.Identity allows different formats of the Personal Identity Number and all of them are valid, in Authentication it still fails with `Unknown error. Please try again.` and as a response 
```
{
    "personalIdentityNumber": "Invalid PersonalIdentityNumber."
}
``` 
if user provides it with the delimiter.
That becomes a bit confusing for users since our mask kind of demands it: 
<img width="386" alt="image" src="https://user-images.githubusercontent.com/5111043/50970453-b5b13f00-14e1-11e9-9f17-4c4642a9c1e5.png">
Without the delimiter everything is good

**Comment by PeterOrneholm** [comment]
Great find! This is fixed in master, but not in the release available at NuGet (1.0.0-RC1), can you confirm this by trying it out in our sample web in Azure?

https://al-samples-identityserver.azurewebsites.net

From the start we have had support for the format presented in the "placeholder" (YYMMDD-NNNN), but not combining the 12 digit version with a delimiter (YYYYMMDD-NNNN). But this was fixed in 1.0.0 of https://github.com/ActiveLogin/ActiveLogin.Identity, but RC1 of this repo pointed to RC1 of https://github.com/ActiveLogin/ActiveLogin.Identity where it was not fixed.

So, I'll close this as it's already fixed in code, just not released. Please confirm that it works for you as well :)

**Comment by nikolaykrondev** [comment]
You are right, too many testing with actual packages, not with master, so I got lost a bit :)
